nemo/collections/llm/gpt/model/llama.py

```diff
@@ -56,7 +56,6 @@ class LlamaConfig(GPTConfig):
     persist_layer_norm: bool = True
     bias_dropout_fusion: bool = True
     apply_rope_fusion: bool = True
-    cross_entropy_loss_fusion: bool = False
 
```

Comment:
wouldnt this disable this for pretraining + SFT?

Reply:
Yep-- I've removed this line to allow the default cross_entropy_loss_fusion=True in GPTConfig to propagate down, while the recipes for llama3 will set it False if LoRA is used.